src/sentry/web/frontend/setup_wizard.py

```diff
@@ -109,11 +112,11 @@ def get(self, request: HttpRequest, wizard_hash) -> HttpResponseBase:
                     default_cache.set(cache_key, cache_data, SETUP_WIZARD_CACHE_TIMEOUT)
 
                     context["enableProjectSelection"] = False
-                    return render_to_response("sentry/setup-wizard.html", context, request)
 
-        context["enableProjectSelection"] = True
-        return render_to_response("sentry/setup-wizard.html", context, request)
+        response = render_to_response("sentry/setup-wizard.html", context, request)
+        return apply_cors_headers(request, response, allowed_methods=self._allowed_methods())
```

Comment:
You shouldn't need to call this explicitly if you have applied the decorator to endpoint method.

Reply:
Ah, I missed that the decorator already applies this - thought it only allows the OPTIONS requests.